### ISSUE TITLE
testunit fails with FileMotFoundError errors

### ISSUE THREAD

**Issue by nunotexbsd** [Diaoul/enzyme#44]
FreeBSD, python3.11, poudriere pkg framework

Having this errors on test unit, any hints how to fixed it?

Thanks

```
tests/test_mkv.py::test_test8 PASSED                                                                                                                                                                         [ 63%]
tests/test_parsers.py::EBMLTestCase::test_parse_full FAILED                                                                                                                                                  [ 72%]
tests/test_parsers.py::EBMLTestCase::test_parse_ignore_element_names FAILED                                                                                                                                  [ 81%]
tests/test_parsers.py::EBMLTestCase::test_parse_ignore_element_types FAILED                                                                                                                                  [ 90%]
tests/test_parsers.py::EBMLTestCase::test_parse_max_level FAILED                                                                                                                                             [100%]

===================================================================================================== FAILURES =====================================================================================================
___________________________________________________________________________________________ EBMLTestCase.test_parse_full ___________________________________________________________________________________________

self = <test_parsers.EBMLTestCase testMethod=test_parse_full>

    def setUp(self):
        self.stream = io.open(os.path.join(TEST_DIR, "test1.mkv"), "rb")
>       with io.open(os.path.join(EBML_VALIDATION_DIR, "test1.mkv.yml"), "r") as yml:
E       FileNotFoundError: [Errno 2] No such file or directory: '/wrkdirs/usr/ports/multimedia/py-enzyme/work-py311/enzyme-0.5.1/tests/parsers/ebml/test1.mkv.yml'

tests/test_parsers.py:27: FileNotFoundError
___________________________________________________________________________________ EBMLTestCase.test_parse_ignore_element_names ___________________________________________________________________________________

self = <test_parsers.EBMLTestCase testMethod=test_parse_ignore_element_names>

    def setUp(self):
        self.stream = io.open(os.path.join(TEST_DIR, "test1.mkv"), "rb")
>       with io.open(os.path.join(EBML_VALIDATION_DIR, "test1.mkv.yml"), "r") as yml:
E       FileNotFoundError: [Errno 2] No such file or directory: '/wrkdirs/usr/ports/multimedia/py-enzyme/work-py311/enzyme-0.5.1/tests/parsers/ebml/test1.mkv.yml'

tests/test_parsers.py:27: FileNotFoundError
___________________________________________________________________________________ EBMLTestCase.test_parse_ignore_element_types ___________________________________________________________________________________

self = <test_parsers.EBMLTestCase testMethod=test_parse_ignore_element_types>

    def setUp(self):
        self.stream = io.open(os.path.join(TEST_DIR, "test1.mkv"), "rb")
>       with io.open(os.path.join(EBML_VALIDATION_DIR, "test1.mkv.yml"), "r") as yml:
E       FileNotFoundError: [Errno 2] No such file or directory: '/wrkdirs/usr/ports/multimedia/py-enzyme/work-py311/enzyme-0.5.1/tests/parsers/ebml/test1.mkv.yml'

tests/test_parsers.py:27: FileNotFoundError
________________________________________________________________________________________ EBMLTestCase.test_parse_max_level _________________________________________________________________________________________

self = <test_parsers.EBMLTestCase testMethod=test_parse_max_level>

    def setUp(self):
        self.stream = io.open(os.path.join(TEST_DIR, "test1.mkv"), "rb")
>       with io.open(os.path.join(EBML_VALIDATION_DIR, "test1.mkv.yml"), "r") as yml:
E       FileNotFoundError: [Errno 2] No such file or directory: '/wrkdirs/usr/ports/multimedia/py-enzyme/work-py311/enzyme-0.5.1/tests/parsers/ebml/test1.mkv.yml'

tests/test_parsers.py:27: FileNotFoundError
=========================================================================================== 4 failed, 7 passed in 0.17s ============================================================================================
*** Error code 1

Stop.
```

**Comment by Diaoul** [comment]
0.5.1 has packaging issues, try 0.5.2

**Comment by nunotexbsd** [comment]
@Diaoul 

Fixed in 0.5.2.
Sorry for the noise.

Thanks
